packages/js/src/first-time-configuration/first-time-configuration-steps.js

```diff
@@ -296,9 +297,33 @@ async function saveFinishedSteps( finishedSteps ) {
  *
  * @returns {WPElement} Finish step.
  */
-const FinishStep = () => <Fragment>
-	<p className="yst-mb-6">You have finished all the things, yay!</p>
-</Fragment>;
+const FinishStep = () => {
+	/**
+	 * Goes to the Dashboard tab.
+	 *
+	 * @param {Event} event The click event.
+	 *
+	 * @returns {void}
+	 */
+	function goToSEODashboard( event ) {
```

Comment:
perhaps we can define this function outside, as it has a global scope-ishness to it. Or inside a usecallback.